flags/eigendaflags/cli.go

```diff
@@ -209,8 +210,13 @@ func validateConfirmationFlag(val string) error {
 		return fmt.Errorf("confirmation-depth must be either 'finalized' or a number, got: %s", val)
 	}
 
-	if depth >= 64 {
-		log.Printf("Warning: confirmation depth set to %d, which is > 2 epochs (64). Consider using 'finalized' instead.\n", depth)
+	if depth >= uint64(consts.EthHappyPathFinalizationDepthBlocks) {
+		// We keep this low (<128) to avoid requiring an archive node (see how this is used in CertVerifier).
+		// Note: assuming here that no sane person would ever need to set this to a number to something >64.
+		// But perhaps someone testing crazy reorg scenarios where finalization takes >2 epochs might want to set this to a higher number...?
+		// Let's deal with that case if and when it comes up (ideally never). Do keep in mind if you ever change this
+		// that it might affect a LOT of validators on your rollup who would now need an archival node.
```

Comment:
knit - feel like the first two comments adequately describe the methodology. The latter 3 increase cognitive complexity imo

Reply:
Overall disagree. Don't think we should be looking to reduce cognitive complexity of comments on flags that are read like one per year, at most. If anything, the person who will want to change that flag value will definitely appreciate having as much context as possible.
Did cleanup the comment though, let me know what you think: [9c43884](https://github.com/Layr-Labs/eigenda-proxy/pull/241/commits/9c438846c3f784b68a640ec1500e4dc919495c20)